packages/@tinacms/cli/src/cmds/forestry-migrate/index.ts

```diff
@@ -110,151 +110,160 @@ const generateCollectionFromForestrySection = (
 ) => {
   if (section.read_only) return
 
-  {
-    // TODO: What should we do with read only sections?
-    if (section.read_only) return
+  // TODO: What should we do with read only sections?
+  if (section.read_only) return
 
-    const baseCollection: Omit<Collection, 'fields' | 'templates'> = {
-      label: section.label,
-      name: stringifyLabel(section.label),
-      path: section.path,
-    }
-    // Add include and exclude if they exist
-    if (section.match) {
-      baseCollection.match = {
-        ...(baseCollection?.match || {}),
-        include: transformForestryMatchToTinaMatch(section.match),
-      }
+  const baseCollection: Omit<Collection, 'fields' | 'templates'> = {
+    label: section.label,
+    name: stringifyLabel(section.label),
+    path: section.path || '/',
```

Comment:
Collections with both an empty string, and path: "/" seems to result in the same error FYI, so I don't know if this will fix the underlying issue

Reply:
Fixed to ignore section like this.